backend/services/inventory/tests/tests/test_api_internal.py

```diff
@@ -12,74 +12,98 @@
 #    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
+
 import os
-from common import (
-    internal_client,
-    management_client,
-    mongo,
-    clean_db,
-    inventory_attributes,
-)
-import bravado
-import pytest
 
+import openapi_client as oas
+import openapi_client.exceptions as api_exceptions
 
-class TestInternalApiTenantCreate:
-    def test_create_ok(self, internal_client, clean_db):
+from client import make_authenticated_client
 
-        _, r = internal_client.create_tenant("foobar")
-        assert r.status_code == 201
+
+class TestInternalApiTenantCreate:
+    def test_create_ok(self, clean_db):
+        internal_client = oas.InventoryInternalApi()
+        rsp = internal_client.create_tenant_with_http_info(
+            oas.TenantNew(tenant_id="foobar")
+        )
+        assert rsp.status_code == 201
```

Comment:
I dont get it, how is that possible that in that case the generator returns the response, which is never the case. I am lost.